### ISSUE TITLE
Feature Request: Add DynamoDB Target to Glue Crawler Resource

### ISSUE THREAD

**Issue by bflad** [terraform-providers/terraform-provider-aws#5151]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Glue has recently added the ability for crawlers to target DynamoDB tables into the Glue SDK. Its likely to be announced on the blog shortly.

### New or Affected Resource(s)

* `aws_glue_crawler`

### Potential Terraform Configuration

```hcl
# Implementation may change during development
resource "aws_glue_crawler" "example" {
  # ... other configuration ...
  dynamodb_target {
    path = "tablename"
  }
}
```

### References

* https://docs.aws.amazon.com/sdk-for-go/api/service/glue/#CrawlerTargets

**Comment by bflad** [comment]
Pull request submitted: #5152

**Comment by bflad** [comment]
This has been released in [version 1.30.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1300-august-02-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
